examples/stocks/lib/stock_home.dart

```diff
@@ -202,11 +201,7 @@ class StockHomeState extends State<StockHome> {
         Navigator.of(context).pushNamed('/stock/${stock.symbol}', mostValuableKeys: mostValuableKeys);
       },
       onShow: (Stock stock, Key arrowKey) {
-        showBottomSheet(
-          placeholderKey: _bottomSheetPlaceholderKey,
-          context: context,
-          child: new StockSymbolBottomSheet(stock: stock)
-        );
+        scaffoldKey.currentState.showBottomSheet(new StockSymbolBottomSheet(stock: stock));
```

Comment:
How do you mean?

On Thu, Nov 19, 2015 at 6:01 PM Adam Barth <notifications@github.com> wrote:

> In examples/stocks/lib/stock_home.dart
> <https://github.com/flutter/flutter/pull/508#discussion_r45428678>:
>
> > @@ -202,11 +201,7 @@ class StockHomeState extends State<StockHome> {
> >          Navigator.of(context).pushNamed('/stock/${stock.symbol}', mostValuableKeys: mostValuableKeys);
> >        },
> >        onShow: (Stock stock, Key arrowKey) {
> > -        showBottomSheet(
> > -          placeholderKey: _bottomSheetPlaceholderKey,
> > -          context: context,
> > -          child: new StockSymbolBottomSheet(stock: stock)
> > -        );
> > +        scaffoldKey.currentState.showBottomSheet(new StockSymbolBottomSheet(stock: stock));
>
> This should probably take a builder to avoid orphaning the widget.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/flutter/flutter/pull/508/files#r45428678>.
>